### ISSUE TITLE
Flagged for PVP duration

### ISSUE THREAD

**Issue by satskilla** [goodmorningbitcoin/tangerine#2]
Upon entering a PVP area, players should remain flagged for PVP for X amount of time (30secs? 60secs? 90secs? 120secs?) after leaving the PVP zone.
This would help prevent players taking advantage of the PVP zones and removing their ability to step in and out of the zone quickly.

**Comment by patrickulrich** [comment]
Right now there's currently a 10 second delay for each of the monuments that have it enabled. 

https://github.com/rustysats/tangerine/blob/main/plugins/config/DynamicPVP.json

For example:

      "Enable PVP Delay": true,
      "PVP Delay Time": 10.0,
      
Should we move the server to having a default 30 second PVP delay across events including RB, Sputnik & Convoy?

**Comment by satskilla** [comment]
I think 30 seconds is pretty conservative and an improvement to 10 seconds.

Is there a visual indicator that can be shown so players still know that they are flagged for PVP? I'm not seeing it in the configuration, but may be blind :)

**Comment by patrickulrich** [comment]
I don't believe dynamicPVP has a feature for flagging. I was able to find this as a plugin we could add in. I'm not sure how it'd support Sputnik or Convoy but lists support for dynamicPVP and RB. https://umod.org/plugins/zone-pvx-info
